src/backend/views/components/sidebar.twig

```diff
@@ -1,55 +1,52 @@
-
 <div data-module="sidebar" class="docs-sidebar">
 
-  <div class="docs-sidebar__toggler">
-    {{ svg('menu') }} Table of contents
-  </div>
+	<div class="docs-sidebar__toggler">
+		{{ svg('menu') }}
```

Comment:
Looks like you have a wrong indentation config in you IDE. We're using two spaces insted of tabulation